pkg/host/networkagent.go

```diff
@@ -0,0 +1,86 @@
+package host
```

Comment:
Might want to add the copyright boilerplate in all the new files.

Reply:
Thanks, it was missing from a bunch of files, I added it everywhere.